src/linear_algebra/affine_space/affine_equiv.lean

```diff
@@ -160,6 +160,17 @@ protected lemma bijective (e : P₁ ≃ᵃ[k] P₂) : bijective e := e.to_equiv.
 protected lemma surjective (e : P₁ ≃ᵃ[k] P₂) : surjective e := e.to_equiv.surjective
 protected lemma injective (e : P₁ ≃ᵃ[k] P₂) : injective e := e.to_equiv.injective
 
+/-- Bijective affine maps are affine isomorphisms. -/
+@[simps]
+noncomputable def of_bijective {φ : P₁ →ᵃ[k] P₂} (hφ : function.bijective φ) : P₁ ≃ᵃ[k] P₂ :=
+{ linear := linear_equiv.of_bijective φ.linear (φ.linear_bijective_iff.mpr hφ),
+  map_vadd' := λ p v, by simp only [equiv.to_fun_as_coe, equiv.coe_fn_mk, equiv.of_bijective_apply,
+                                    affine_map.map_vadd, linear_equiv.of_bijective_apply],
```

Comment:
I would guess this has a one-line proof, probably just
```suggestion
  map_vadd' := φ.map_vadd,
```
or similar

Reply:
Yes, thanks for that one!